controllers/managedfusion_controller.go

```diff
@@ -432,6 +444,11 @@ func (r *ManagedFusionReconciler) reconcilePrometheus() error {
 
 		r.prometheus.Spec = desired.Spec
 		r.prometheus.Spec.ExternalLabels["clusterId"] = string(clusterVersion.Spec.ClusterID)
+		if r.remoteWriteConfig.BearerToken != "" {
+			r.prometheus.Spec.RemoteWrite[0].BearerToken = r.remoteWriteConfig.BearerToken
+		} else {
+			r.prometheus.Spec.RemoteWrite[0] = promv1.RemoteWriteSpec{}
```

Comment:
With this change, remote write gets disabled and there is no error in the Prometheus pod